runtime/vm/jvminit.c

```diff
@@ -2253,6 +2256,45 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				printLockwordWhat(vm);
 			}
 
+			/* Global Lock Reservation is off by default. */
+			vm->enableGlobalLockReservation = 0;
+
+			/* Set default parameters for Global Lock Reservation. */
+			vm->reservedTransitionThreshold = 1;
+			vm->reservedAbsoluteThreshold = 10;
+			vm->minimumReservedRatio = 1024;
+			vm->cancelAbsoluteThreshold = 10;
+			vm->minimumLearningRatio = 256;
+
+			argIndex = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXNOGLOBALLOCKRESERVATION, NULL);
+			argIndex2 = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXGLOBALLOCKRESERVATION, NULL);
+
+			if (argIndex2 >= 0 && argIndex2 > argIndex) {
+				/* Global Lock Reservation is currently only supported on Power. */
+#if (defined(AIXPPC) || defined(LINUXPPC))
+				vm->enableGlobalLockReservation = 1;
+#endif
```

Comment:
The outermost parentheses are not necessary here, but `#endif` lines should have a comment showing the matching `#if` expression.

Reply:
I will remove the parentheses and add the comment. I'll also add similar comments to the `#endif` in `j9nonbuilder.h` and `ObjectMonitor.cpp` that I added.